server/core/src/main/scala/sttp/tapir/server/metrics/Metric.scala

```diff
@@ -63,4 +63,25 @@ object MetricLabels {
       }
     )
   )
+
+  /** Default labels for OpenTelemetry-compliant metrics, as recommended here:
+    * https://opentelemetry.io/docs/specs/semconv/http/http-metrics/#http-server
+    *
+    *   - `http.request.method` - HTTP request method (e.g., GET, POST).
+    *   - `path` - The request path or route template.
+    *   - `http.response.status_code` - HTTP response status code (200, 404, etc.).
+    */
+  lazy val OpenTelemetryAttributes: MetricLabels = MetricLabels(
```

Comment:
actually ... thinking about this, this should go to the OpenTelemetry module, as it's OT-specific. Probably as a value in `OpenTelemetryMetrics`

Reply:
But It seems good all the relate of metrics are in same place for better readability

Reply:
Or rather OTEL-specific